components/script/dom/document.rs

```diff
@@ -1544,9 +1544,7 @@ impl Document {
         self.browsing_context.is_none() || !url_has_network_scheme(&self.url)
     }
 
-    pub fn nodes_from_point(&self, page_point: &Point2D<f32>) -> Vec<UntrustedNodeAddress> {
-        assert!(self.GetDocumentElement().is_some());
-
+     pub fn nodes_from_point(&self, page_point: &Point2D<f32>) -> Vec<UntrustedNodeAddress> {
```

Comment:
Indentation fail. Just press backspace once, so the `pub fn` lines up with the previous line's `}`